sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -442,43 +337,14 @@ export function isPartitionManager(
  * @internal
  * @ignore
  */
-function isSubscriptionOptions(possible: SubscriptionOptions | string | PartitionManager) : possible is SubscriptionOptions{
-  return typeof (possible as SubscriptionOptions).processEvents === "function";
+function isSubscriptionEventHandlers(possible: any | SubscriptionEventHandlers): possible is SubscriptionEventHandlers {
+  return typeof (possible as SubscriptionEventHandlers).processEvent === "function";
 }
 
-class SimplePartitionCheckpointer implements PartitionCheckpointer, PartitionContext {
-  private _eTag: string = "";
-
-  constructor(private _manager: PartitionManager, private _processor: PartitionProcessor, public eventHubName: string, public consumerGroupName: string, public partitionId: string, public fullyQualifiedNamespace: string) {   }
-
-  async updateCheckpoint(
-    eventData: ReceivedEventData
-  ): Promise<void>;
-  async updateCheckpoint(
-    sequenceNumber: number,
-    offset: number
-  ): Promise<void>;
-  async updateCheckpoint(
-    eventDataOrSequenceNumber: ReceivedEventData | number,
-    offset?: number
-  ): Promise<void> {
-    const checkpoint: Checkpoint = {
-      fullyQualifiedNamespace: this._processor.fullyQualifiedNamespace!,
-      eventHubName: this._processor.eventHubName!,
-      consumerGroupName: this._processor.consumerGroupName!,
-      ownerId: this._processor.eventProcessorId!,
-      partitionId: this._processor.partitionId!,
-      sequenceNumber:
-        typeof eventDataOrSequenceNumber === "number"
-          ? eventDataOrSequenceNumber
-          : eventDataOrSequenceNumber.sequenceNumber,
-      offset:
-        typeof offset === "number"
-          ? offset
-          : (eventDataOrSequenceNumber as ReceivedEventData).offset,
-      eTag: this._eTag
-    };
-
-    this._eTag = await this._manager.updateCheckpoint(checkpoint);
+function getOwnerLevel(options?: SubscriptionOptions) : number {
+  if (options && options.ownerLevel) {
```

Comment:
Fixed this - the caller owns the default value (0 or undefined). 

Out of our subscribe overloads only one requires an explicit partition manager, so that defaults to 0. The others default to undefined.